### ISSUE TITLE
<vaadin-form-item> changing the labels style

### ISSUE THREAD

**Issue by limonte** [vaadin/vaadin-form-layout#75]
Live demo: https://jsfiddle.net/6r5htd05/281/

All lovely styles for the label - colors, font-family, etc. are lost:

![image](https://user-images.githubusercontent.com/6059356/39620065-db033f8e-4fc4-11e8-8c34-ee9d849f6522.png)

Is that how it should be by default?

**Comment by jouni** [comment]
You don’t seem to be importing vaadin-form-item.html

**Comment by limonte** [comment]
Oops, stupid me. Fixed now. 

There's still a small issue with colors:

- dark grey color isn't applied on hover
- blue color isn't applied on activate

**Comment by jouni** [comment]
Yeah, the interactive states are missing. My feeling is that the easiest way to support that would be something like I described in https://github.com/vaadin/vaadin-checkbox/issues/27#issuecomment-384952265

**Comment by web-padawan** [comment]
Updated the issue to reflect the actual problem.
